contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -260,9 +261,17 @@ var BaseWorkspaceView = BaseView.extend({
 	},
 	handle_published:function(collection){
 		this.reload_ancestors(collection);
-		var staticModal = require('edit_channel/information/views');
-		new staticModal.PublishedModalView({channel_id: window.current_channel.id});
+		var self = this;
+		window.current_channel.fetch({
+			success: function(channel){
+				var new_channel = new Models.ChannelCollection()
+				new_channel.reset(channel.toJSON());
+				var staticModal = require('edit_channel/information/views');
+				new staticModal.PublishedModalView({primary_token: window.current_channel.attributes.primary_token});
```

Comment:
Better to use window.current_channel.get("primary_token") https://stackoverflow.com/questions/9763560/backbone-js-difference-between-getters-vs-direct-access-of-model-attributes